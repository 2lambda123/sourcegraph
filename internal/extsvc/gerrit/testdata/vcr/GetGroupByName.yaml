---
version: 1
interactions:
- request:
    body: ""
    form: {}
    headers: {}
    url: https://gerrit.sgdev.org/a/groups/Sourcegraph
    method: GET
  response:
    body: |
      )]}'
      {
        "url": "#/admin/groups/uuid-40bb3b32f9da57a0697285daabb777965720b96e",
        "options": {
          "visible_to_all": true
        },
        "group_id": 3,
        "owner": "Sourcegraph",
        "owner_id": "40bb3b32f9da57a0697285daabb777965720b96e",
        "created_on": "2020-11-30 16:51:34.000000000",
        "id": "40bb3b32f9da57a0697285daabb777965720b96e",
        "name": "Sourcegraph"
      }
    headers:
      Cache-Control:
      - no-cache, no-store, max-age=0, must-revalidate
      Cf-Cache-Status:
      - DYNAMIC
      Cf-Ray:
      - 7218f6618f3cf216-ORD
      Content-Disposition:
      - attachment
      Content-Type:
      - application/json;charset=utf-8
      Date:
      - Sun, 26 Jun 2022 21:16:09 GMT
      Expect-Ct:
      - max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
      Expires:
      - Mon, 01 Jan 1990 00:00:00 GMT
      Pragma:
      - no-cache
      Server:
      - cloudflare
      Strict-Transport-Security:
      - max-age=15724800; includeSubDomains
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - DENY
    status: 200 OK
    code: 200
    duration: ""
