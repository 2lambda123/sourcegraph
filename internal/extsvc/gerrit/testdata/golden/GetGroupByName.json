{
  "id": "40bb3b32f9da57a0697285daabb777965720b96e",
  "group_id": 3,
  "name": "Sourcegraph",
  "description": "",
  "created_on": "2020-11-30 16:51:34.000000000",
  "owner": "Sourcegraph",
  "owner_id": "40bb3b32f9da57a0697285daabb777965720b96e"
 }